README.md

```diff
@@ -109,22 +111,49 @@ To generate your a new key \(default _ed25519_ elliptic curve\):
 gaiacli keys add default
 ```
 
-Next, you will have to enter a passphrase for your key twice. Save the _seed phrase_ in a safe place in case you forget the password.
 
-Now if you check your private keys you will see the key among them:
+Next, you will have to enter a passphrase for your `$KEYNAME` key twice. Save the _seed_ _phrase_ in a safe place in case you forget the password.
```

Comment:
`$KEYNAME` is never setup before this point. 

Generally people do not understand environment variables very well. Let's stick to the `<your_name>` syntax instead everywhere.